pkg/nuctl/test/deploy_test.go

```diff
@@ -124,6 +127,64 @@ func (suite *DeployTestSuite) TestDeployWithMetadata() {
 	suite.Require().Contains(suite.outputBuffer.String(), "0099887766")
 }
 
+func (suite *DeployTestSuite) TestDeployDirectory() {
+	imageName := fmt.Sprintf("nuclio/deploy-test-%s", xid.New().String())
+
+	err := suite.ExecuteNutcl([]string{"deploy", "reverser", "--verbose", "--no-pull"},
+		map[string]string{
+			"path":    path.Join(suite.GetFunctionsDir(), "common", "reverser", "golang"),
+			"image":          imageName,
```

Comment:
Not properly formatted with `gofmt` - @pavius, is this supposed to pass the linter check?